### DISCUSSION TITLE
init: MYDS Discussions!

### DISCUSSION THREAD

**Discussion by irfancoder** [govtechmy/myds#93] [Announcements]
## 👋 Welcome to MYDS!

MYDS is a design system developed by Govtech Nucleus Unit (GNU) under Kementerian Digital. Our goal for MYDS is to be used by developers far and wide when developing Malaysian Govt web products, thereby, improving the average Malaysian experience online. 

Best part of all, we are open source! So, we invite you to be part of MYDS as a contributor, thinker, or as a user. 

Use the Discussion space to:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

## GNU who?

GNU is a unit comprised of software engineers, designers, data scientists under Kementerian Digital. The team has been around since the days of COVIDNOW and has worked on several high impact products for the nation such as, OpenDOSM, data.gov.my and many more. Our aim is to continue to shake up how governments deliver products (wrt quality) until it has become a common sight, with the public expect nothing less, but excellence. Nice to meet you!

## Getting started

To get started, comment below with an introduction of yourself and tell us about how you plan to contribute to MYDS!

**Comment by txz-169** [comment]
Hi! I'm passionate about good UI/UX designs. I have zero programming knowledge, so I'll be contributing by reporting issues and providing suggestions. Looking forward to the full launch and end of ugly gov.my websites!

**Comment by acfatah** [comment]
Salam.

I am a Vue.js frontend developer, and I find this project very interesting. I have a question regarding its legal documentation. I noticed that the project is copyrighted by the Ministry of Digital, but I could not find any specific details about its open-source license in the repository or on the website.

Could you clarify the type of open-source license applied to this project? I would also appreciate any information on its licensing terms.

Thank you in advance!

**Reply by alzhahir** [reply]
Salam. I agree with you. The repo does not list any information regarding the license, but the README claims that the project is open source. According to GitHub, [as long as no specific open-source license is defined, default copyright laws apply](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository). Hopefully the GNU team would address this as soon as possible.
